kustomization/components/kube-vip/rbac.yml

```diff
@@ -7,8 +9,6 @@ metadata:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  annotations:
-    rbac.authorization.kubernetes.io/autoupdate: "true"
```

Comment:
This is already the default behavior per https://kubernetes.io/docs/reference/access-authn-authz/rbac/#auto-reconciliation